src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2355,6 +2277,7 @@ because xbuild doesn't support framework reference assemblies.
 <PropertyGroup>
   <_GeneratePackageManagerJavaDependsOn>
     _GenerateJavaStubs;
+    $(_AfterGenerateJavaStubs);
```

Comment:
Do the designer targets need any changes? Does it need to use `$(_AfterGenerateJavaStubs)` in some way?

Reply:
it will probably. given they use Aapt2 as well.